### ISSUE TITLE
Batoto & mirrors - Error while getting pages list

### ISSUE THREAD

**Issue by BaratDaya** [RedSquirrel87/manga-downloader#823]
Hi, just want to report the problem I have, every time I tried to download from batoto and it's mirrors.
It always shows progress error, and the log says:
**Error while getting pages list!**

It happens only through batoto, and unfortunately, the manga I want to download is available only at that site. I tried to restart it, and change vpn, still not working.
[jvm_log.txt](https://github.com/RedSquirrel87/manga-downloader/files/14812350/jvm_log.txt)

I attached the log also.

**Comment by RedSquirrel87** [comment]
Yes, batoto is currently broken on public release but it has already been fixed in nightly builds (and so it will be fixed in next public release). 
Duplicate of #818
